### ISSUE TITLE
What is the objective when pretraining?

### ISSUE THREAD

**Issue by Young973** [YuanGongND/ast#107]
TBH, I'm a little confused about what is the objective when pretraining with AST? It seems it is not indicated in the paper. BTW, when pretraining SSAST discriminative objective is the classification with InfoNCE and generative objective is reconstruction. But what is it in AST?

**Comment by YuanGongND** [comment]
hi there,

It is just ImageNet pretraining.

I.e., using ImageNet pretrained DeiT as the initial weight for AST. 

https://github.com/YuanGongND/ast/blob/31088be8a3f6ef96416145c4b8d43c81f99eba7a/src/models/ast_models.py#L60-L68

-Yuan

**Comment by YuanGongND** [comment]
Some modification is needed. See https://github.com/YuanGongND/ast/blob/master/src/models/ast_models.py.

**Comment by YuanGongND** [comment]
If you mean audio domain pretraining, that is just train AST on AudioSet (based on ImageNet initialization) with BCE loss for classification task. You can then take the model for other audio tasks (e.g., for ESC-50).
